future-apps/payroll/contracts/Payroll.sol

```diff
@@ -461,6 +523,12 @@ contract Payroll is AragonApp { //, IForwarder { // makes coverage crash (remove
         nextEmployee++;
     }
 
+    function _addAccruedValue(uint128 employeeId, uint256 amount) internal {
+        employees[employeeId].accruedValue = employees[employeeId].accruedValue.add(amount);
```

Comment:
Hm, interesting. You are right, but I guess that even setting the `accruedValue` to such an amount (`2^128`) would still block the employee in most cases as there wouldn't be enough funds to execute `payday`. I guess the best solution would be to allow to withdraw as much as possible up to owed amount, but that would make logic even way more complex, taking into account the multi-currency support.

Reply:
I was thinking about limiting it up to the yearly salary. I think this function was meant to be used for things like expenses, bonuses, etc. They are usually less than one year of salary, but even if it's more but not much more, it could be done using several transactions. So it would cover most of the use case being quite safe... The problem is that I realized the same problem happens with the salary itself.